### ISSUE TITLE
[Quest] [Northrend] The Plains Of Nassam 

### ISSUE THREAD

**Issue by Broskheim** [dalaranwow/dalaran-wow#5656]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Where: Borean Tundra
When: My issue occurred 1/1/19 at 1:10 AM
What: The tanks are very buggy.  Sometimes they will kick me out unexpectedly, and when I do manage to get to the quest area on the plain, none of my abilities will damage the enemies or pick up the wounded allies, making this quest impossible to complete.

**Expected behaviour**: Tell us what should happen instead
The abilities should damage the enemies and the allies should be able to be picked up.

**Steps to reproduce the problem**:

1. Accept the Plains of Nassam Quest
2. Board the tank
3. Drive to the quest area on the plains
4. Attempt to damage enemies

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
